packages/compiler-cli/src/transformers/program.ts

```diff
@@ -7,26 +7,28 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {AotCompiler, AotCompilerHost, AotCompilerOptions, core, createAotCompiler, EmitterVisitorContext, FormattedMessageChain, GeneratedFile, getParseErrors, isFormattedError, isSyntaxError, MessageBundle, NgAnalyzedFile, NgAnalyzedFileWithInjectables, NgAnalyzedModules, ParseSourceSpan, PartialModule, Position, Serializer, StaticSymbol, TypeScriptEmitter, Xliff, Xliff2, Xmb} from '@angular/compiler';
+import {AotCompiler, AotCompilerOptions, core, createAotCompiler, FormattedMessageChain, GeneratedFile, getParseErrors, isFormattedError, isSyntaxError, MessageBundle, NgAnalyzedFileWithInjectables, NgAnalyzedModules, ParseSourceSpan, PartialModule, Serializer, Xliff, Xliff2, Xmb} from '@angular/compiler';
 import * as fs from 'fs';
 import * as path from 'path';
 import * as ts from 'typescript';
 
-import {translateDiagnostics, TypeCheckHost} from '../diagnostics/translate_diagnostics';
-import {createBundleIndexHost, MetadataCollector, ModuleMetadata} from '../metadata';
+import {translateDiagnostics} from '../diagnostics/translate_diagnostics';
+import {createBundleIndexHost, MetadataCollector} from '../metadata';
+import {isAngularCorePackage} from '../ngtsc/core/src/compiler';
```

Comment:
ngtsc tries to ensure that all imports go through the barrel `index.ts` files - so this should be:
```suggestion
import {isAngularCorePackage} from '../ngtsc/core';
```
You might need to add a re-export to make that work.